src/org/jitsi/impl/neomedia/rtp/translator/SimulcastController.java

```diff
@@ -147,6 +147,20 @@ public long getTargetSSRC()
      */
     public boolean accept(byte[] buf, int off, int len)
     {
+        int targetIndex = bitstreamController.getTargetIndex()
+            , currentIndex = bitstreamController.getCurrentIndex();
+
+        if (targetIndex < 0 && currentIndex > -1)
+        {
+            synchronized (this)
```

Comment:
This is for the case where we want to suspend a stream. We handle stream suspension just like a simulcast switch to another RTP stream, so we get keyframe management for free (i.e. when a stream gets re-activated, it needs to start with a key frame). This is exactly the purpose of the SimulcastController and it would be a pity to re-implement this logic in the BitstreamController.